packages/editor/src/extensions/index.ts

```diff
@@ -24,6 +24,9 @@ export const editorExtensions = (
   callback: (resultMap: Map<string, ValidationResult>) => void,
 ) => [
   Document,
+  Placeholder.configure({
+    placeholder: 'Start met typen...',
+  }),
```

Comment:
Is de placeholder toegankelijk? Als we dat niet zeker weten zou ik het even uitstellen en op de backlog zetten.